### ISSUE TITLE
chore: export CustomizeIconType type from index.d.ts

### ISSUE THREAD

**Issue by Zhengqbbb** [Renovamen/oh-vue-icons#18]
### Why:
My icons array is not well supported and type checked when I use typescript
```ts
const icons = [
  {
    name: "mail",
    width: 24,
    height: 24,
    d: "M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm-1 14H5c-.55 0-1-.45-1-1V8l6.94 4.34c.65.41 1.47.41 2.12 0L20 8v9c0 .55-.45 1-1 1zm-7-7L4 6h16l-8 5z"
  }
];
addIcons(...icons);
```

### Expect
```ts
import type { CustomizeIconType } from "oh-vue-icons";
const icons: CustomizeIconType[] = [
  {
    name: "mail",
    width: 24,
    height: 24,
    d: "M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm-1 14H5c-.55 0-1-.45-1-1V8l6.94 4.34c.65.41 1.47.41 2.12 0L20 8v9c0 .55-.45 1-1 1zm-7-7L4 6h16l-8 5z"
  }
];
addIcons(...icons);
```

**Comment by Zhengqbbb** [comment]
Need to post a version on npm~~~👀👀👀

**Comment by Renovamen** [comment]
Released (v1.0.0-rc2), thanks!
